keyboards/alpine65/keymaps/default/keymap.c

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2012,2013 Jun Wako <wakojun@gmail.com>
```

Comment:
I guess you forgot to update the copyright

Reply:
Committed in 3240ab0